generators/server/templates/src/main/java/package/config/_LoggingConfiguration.java

```diff
@@ -74,24 +76,24 @@
     <%_ if (serviceDiscoveryType === "consul") { _%>
 
     private final ConsulRegistration consulRegistration;
-    <%_ } _%>
 
     private final String version;
+    <%_ } _%>
 
     private final JHipsterProperties jHipsterProperties;
 
     public LoggingConfiguration(@Value("${spring.application.name}") String appName, @Value("${server.port}") String serverPort,
-        <% if (serviceDiscoveryType === "eureka") { %>EurekaInstanceConfigBean eurekaInstanceConfigBean,<% } %><% if (serviceDiscoveryType === "consul") { %> ConsulRegistration consulRegistration,<% } %> JHipsterProperties jHipsterProperties, @Value("${info.project.version}") String version) {
+        <% if (serviceDiscoveryType === "eureka") { %>EurekaInstanceConfigBean eurekaInstanceConfigBean,<% } %><% if (serviceDiscoveryType === "consul") { %> ConsulRegistration consulRegistration, @Value("${info.project.version}") String version,<% } %> JHipsterProperties jHipsterProperties) {
         this.appName = appName;
         this.serverPort = serverPort;
         <%_ if (serviceDiscoveryType === 'eureka') { _%>
         this.eurekaInstanceConfigBean = eurekaInstanceConfigBean;
         <%_ } _%>
         <%_ if (serviceDiscoveryType === 'consul') { _%>
         this.consulRegistration = consulRegistration;
+        this.version = version;
```

Comment:
I don't understand here.
So only 'consul option' will have version?

Reply:
You need to check the code above too:
https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/java/package/config/_LoggingConfiguration.java#L116-L121

Reply:
sorry you are correct, I'm changing this, the variable is only unused for monoliths